frontends/web/src/routes/account/account.tsx

```diff
@@ -123,8 +124,17 @@ export function Account({
 
   useEffect(() => {
     maybeCheckBitsuranceStatus();
-    getConfig().then(({ backend }) => setUsesProxy(backend.proxy.useProxy));
-  }, [maybeCheckBitsuranceStatus]);
+    getConfig().then(({ backend }) => {
+      setUsesProxy(backend.proxy.useProxy);
+      if (account) {
+        if (backend[account.coinCode]) {
+          setBlockExplorerTxPrefix(backend[account.coinCode].blockExplorerTxPrefix);
+        } else {
```

Comment:
this is redundant as we check `blockExplorerTxPrefix` undefined on line 337. Should I remove the else branch ?